tsconfig.json

```diff
@@ -15,7 +15,8 @@
     "allowJs": false,
     "declaration": true,
     "noEmitOnError": true,
-    "allowSyntheticDefaultImports": true
+    "allowSyntheticDefaultImports": true,
+    "baseUrl": "./src"
```

Comment:
I understand why you added this but I don't think we can, see #717.

Reply:
I removed, thanks!